### PR TITLE
Update zh-Hans.json

### PR DESCRIPTION
**Pull request by uinz**
"登录" is better than "登陆"

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [x] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: packages/strapi-admin/admin/src/translations/zh-Hans.json [hunk 1]
```diff
@@ -137,7 +137,7 @@
   "Auth.advanced.allow_register": "",
   "Auth.form.button.forgot-password": "发送电子邮件",
   "Auth.form.button.forgot-password.success": "再次发送",
-  "Auth.form.button.login": "登陆",
+  "Auth.form.button.login": "登录",
```
Comment (Xiaohanxu1996) [inline packages/strapi-admin/admin/src/translations/zh-Hans.json]: In Chinese, ”登录“ means login, while "登陆" means disembark. "登录" is more accurate.

File: packages/strapi-admin/admin/src/translations/zh-Hans.json [hunk 2]
```diff
@@ -176,6 +176,6 @@
   "Auth.form.register.username.placeholder": "John Doe",
   "Auth.header.register.description": "要完成安装并保护您的应用程序，请通过输入必要的信息来创建第一个用户（root管理员）。",
   "Auth.link.forgot-password": "忘记密码了吗？",
-  "Auth.link.ready": "准备好登陆？",
+  "Auth.link.ready": "准备好登录？",
```
Comment (Xiaohanxu1996) [inline packages/strapi-admin/admin/src/translations/zh-Hans.json]: In Chinese, ”登录“ means login, while "登陆" means disembark. "登录" is more accurate.

**Comment by ly525** [comment]
agree with this

all the Chinese top 10 websites use  “登录”


百度http://baidu.com —登录
腾讯网http://qq.com —一键登录
淘宝http://taobao.com —登录
新浪http://sina.com.cn —登录
网易http://163.com —登录
搜搜http://soso.com —登录
搜狐http://soho.com —登录
hao123 http://hao123.com —登录
优酷 http://youku.com —登录
凤凰网http://ifeng.com —登录

stat data source：https://www.alexa.com/topsites/countries/CN

**Comment by whwnow** [comment]
maybe use zh-CN(Simplified Chinese) and zh-TW(Traditional Chinese) is better? This is most often used
ant.design‘s i18n
amazon's i18n

**Pull request merged**